app/views/manage/finishers/_finisher_card.html.haml

```diff
@@ -1,8 +1,11 @@
 - classes ||= []
-%div{:class => [:card, finisher.active_assignments.any? && :assigned ]}
+%div{:class => [:card, finisher.active_assignments.any? && :assigned ].concat(classes)}
   - if finisher.unavailable
     .unavailable
       Unavailable
+  - elsif finisher.has_volunteer_time_off?
+    .px-2.text-white.font-weight-bold.mb-0.text-center{style: 'background-color:#f437ac'}
```

Comment:
I feel like this color definition belongs in the SASS file. Maybe a variable is over engineering it but a class name seems good enough. In-line styles are hard to track down if we ever want to make colors consistent.